app/assets/javascripts/miq_application.js

```diff
@@ -1345,10 +1345,8 @@ function miqSerializeField(element, field_name) {
 }
 
 function miqInitSelectPicker() {
-  $('.selectpicker').selectpicker();
   $('.selectpicker').selectpicker({
-    style: 'btn-info',
-    size: 10
```

Comment:
@hayesr removal of this line here is causing drop down with lots of elements to expand upto the end of screen, see before/after screenshots below:
@epwinchell please comment if this looks ok to you.
before
![before](https://cloud.githubusercontent.com/assets/3450808/16165430/d6a38a6e-34b1-11e6-8f8d-73d87106ec5f.png)

after
![after](https://cloud.githubusercontent.com/assets/3450808/16165432/da774630-34b1-11e6-9d6b-f0fb5f7c4b66.png)